src/base/system_parameter.c

```diff
@@ -2342,11 +2342,9 @@ bool PRM_CDC_LOGGING_DEBUG = false;
 static bool prm_cdc_logging_debug_default = false;
 static unsigned int prm_cdc_logging_debug_flag = 0;
 
-static unsigned int prm_recovery_parallel_count_flag = 0;
-static int prm_recovery_parallel_count_default = 8;
-int PRM_RECOVERY_PARALLEL_COUNT_CURRENT_VALUE = 8;
-static int prm_recovery_parallel_count_upper_value = 32;
-static int prm_recovery_parallel_count_lower_value = 0;
+static unsigned int prm_recovery_parallel_flag = 0;
+static bool prm_recovery_parallel_default = true;
+bool PRM_RECOVERY_PARALLEL_CURRENT_VALUE = true;
```

Comment:
```suggestion
bool PRM_RECOVERY_PARALLEL = true;
```
다른 시스템 파라미터들을 보면, 일반적으로, 파라미터명을 대문자로 표기해서 사용하네요.

Reply:
시스템 파라미터 명을 변경할 때 함께 변경하겠습니다.

Reply:
시스템파라미터 제거했습니다.